stock_report_quantity_by_location/wizards/stock_report_quantity_by_location_prepare.py

```diff
@@ -79,3 +84,40 @@ def _compute_stock_report_by_location(self):
                         }
                     )
         self.env["stock.report.quantity.by.location"].create(vals_list)
+
+    def button_export_html(self):
+        self.ensure_one()
+        action = self.env.ref(
+            "stock_report_quantity_by_location."
+            "action_stock_report_quantity_by_location_html"
+        )
+        vals = action.read()[0]
+        new_context = vals.get("context", {})
+        if isinstance(new_context, str):
+            try:
+                new_context = safe_eval(new_context)
+            except (TypeError, SyntaxError, NameError, ValueError):
+                _logger.warning(
+                    _("Failed context evaluation: %(context)s", context=new_context)
```

Comment:
**suggestion**: let's maintain the same string translation/formatting structure as the other modules

```suggestion
                    _("Failed context evaluation: %(context)s") % {"context": new_context}
```

Reply:
Modified as suggested